src/components/DayPickerNavigation.jsx

```diff
@@ -126,6 +139,11 @@ function DayPickerNavigation({
           styles.DayPickerNavigation__verticalScrollable,
           isDefaultNav && styles.DayPickerNavigation__verticalScrollableDefault,
         ] : []),
+        ...(isBottomNavPosition ? [
+          styles.DayPickerNavigation__bottom,
+          isDefaultNav && styles.DayPickerNavigation__bottomDefault,
+        ] : []),
+        hasCustomStyles && customStyles,
```

Comment:
:-/ this allows full customization of all CSS, which is typically something we've avoided allowing.

cc @majapw @noratarano; do you think there's a way we could solve this with theming, or find a non-flex fallback?

Reply:
ahhh I see 😢 @majapw @noratarano happy to sit down with you in person on Thursday if that's easier to walk through the issue

Reply:
@ljharb I chatted with @noratarano and we feel that using inline styles (renamed to `inlineStyles` vs. `customStyles`) is the best option here given the implementation and timeline constraints. wdyt of leaving this as-is? if you're okay with this I noticed that coverage dropped, I'll make sure to up that percentage by adding more tests.

Reply:
It’s certainly not my call, but i think allowing unrestricted styling is a major philosophical departure from both external (and to a lesser extent, internal) policies thus far.